src/llfs/storage_file_builder.test.cpp

```diff
@@ -241,4 +347,93 @@ TEST_F(StorageFileBuilderTest, WriteReadFile)
   ASSERT_TRUE(recovered_device.ok()) << BATT_INSPECT(recovered_device.status());
 }
 
+//---------------------------------------------------------------------------------------------------
+
+TEST_F(StorageFileBuilderTest, WriteReadManyPackedConfigs)
+{
+  i64 base_file_offset = 0;
+  llfs::StatusOr<llfs::ScopedIoRing> ioring =
+      llfs::ScopedIoRing::make_new(llfs::MaxQueueDepth{1024}, llfs::ThreadPoolSize{1});
+
+  ASSERT_TRUE(ioring.ok()) << BATT_INSPECT(ioring.status());
+
+  const char* const test_file_name = "/tmp/llfs_test_file";
+  std::filesystem::remove(test_file_name);
+  boost::uuids::uuid page_device_uuid;
+
+  llfs::StatusOr<int> test_fd =
+      llfs::create_file_read_write(test_file_name, llfs::OpenForAppend{false});
+  ASSERT_TRUE(test_fd.ok()) << BATT_INSPECT(test_fd.status());
+  {
+    llfs::Status status = llfs::enable_raw_io_fd(*test_fd, /*enabled=*/true);
+    ASSERT_TRUE(status.ok()) << BATT_INSPECT(status);
+  }
+  llfs::IoRingRawBlockFile test_file{llfs::IoRing::File{ioring->get_io_ring(), *test_fd}};
+
+  this->write_rand_data(test_file, /*write_offset=*/base_file_offset,
```

Comment:
If there is a failure here, the test will continue to run from here (after `write_rand_data` returns), even though the test will eventually fail.

Reply:
Function now returns batt::Status and checks prior to continuing.